core/apps/frontend/pages/account_page.py

```diff
@@ -29,6 +29,21 @@ class AccountPage(BasePage):
     from_account_dropdown_items = browser.elements(by.xpath(f"//select[@id='fromAccountId']/option"))
     to_account_dropdown_items = browser.elements(by.xpath(f"//select[@id='toAccountId']/option"))
 
+    bill_pay_option = browser.element(by.xpath("//a[text()='Bill Pay']"))
```

Comment:
It's fine to use text for xpaths here. But pay attention, that it wouldn't work in case of multilanguage sites. In that case you shouldn't rely on text or use (text()='Bill Pay' or text()='Сплатити рахунок' etc). Please pay attention to this